hydrosdk/modelversion.py

```diff
@@ -384,11 +384,13 @@ def from_json(cluster: Cluster, model_version: dict) -> 'ModelVersion':
         model_contract = contract_dict_to_ModelContract(model_version["modelContract"])
 
         # external model deserialization handling
-        if not model_version.get("runtime"):
-            model_version["runtime"] = {}
+        is_external = model_version.get('isExternal', False)
```

Comment:
I am not sure what should be the default behavior in this situation. I suppose Hydrosphere manager **always** returns `isExternal` field. Shouldn't lack of this field be cause for concern? @KineticCookie